fetch.md

```diff
@@ -1,11 +1,204 @@
-# Prerendering fetching modes
+# Prerendering Fetching Modes
 
-Placeholder. TODO.
+## Related Issues and Discussion
```

Comment:
Nit: heading casing is inconsistent. I prefer sentence-case and that's what some of the other documents in the repo do, but it doesn't really matter.

Reply:
Thanks, I went with sentence-case.